Cranky.toml

```diff
@@ -152,6 +156,7 @@ allow = [
 
   # TODO(emilk): enable more of these lints:
   "clippy::cloned_instead_of_copied",
+  "clippy::iter_over_hash_type",
```

Comment:
happens too often? ah well

Reply:
More than twice, at this is too fun to fix this cycle :)